balena-engine/build.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+###
+# Copyright 2019 resin.io
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+set -u
+set -e
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+usage () {
+  echo "Usage: $0" 1>&2
+  echo "" 1>&2
+  echo "Options" 1>&2
+  echo "" 1>&2
+  echo "    -b <base project directory>" 1>&2
+  echo "    -r <architecture>" 1>&2
+  exit 1
+}
+
+ARGV_BASE_DIRECTORY=""
+ARGV_ARCHITECTURE=""
+
+while getopts ":b:r:" option; do
+  case $option in
+    b) ARGV_BASE_DIRECTORY=$OPTARG ;;
+    r) ARGV_ARCHITECTURE=$OPTARG ;;
+    *) usage ;;
+  esac
+done
+
+if [ -z "$ARGV_BASE_DIRECTORY" ] \
+  || [ -z "$ARGV_ARCHITECTURE" ]
+then
+  usage
+fi
+
+GOARM=""
+case $ARGV_ARCHITECTURE in
+  arm5) GOARM=5 ;;
+  arm6) GOARM=6 ;;
+  arm7) GOARM=7 ;;
```

Comment:
Is this required for the concourse to work? I am taking care of any Go related build environment things in the images. Actually arm* and aarch64 builds will run on qemu, we wouldn't really need these...